Makefile

```diff
@@ -52,8 +47,8 @@ clean:
 	rm -f .coverage
 
 all:
-	make -C misc
-	make -C qubes-rpc
+	$(MAKE) -C misc VERSION=$$(< version)
```

Comment:
You already have `$(VERSION)`, you can use that here, instead of re-reading the file.

Reply:
fixed